modules/kubernetes-addons/aws-efs-csi-driver/locals.tf

```diff
@@ -0,0 +1,55 @@
+locals {
+  service_account_name = "efs-csi-sa"
```

Comment:
Change the name to 
```hcl
service_account_name = "efs-csi-controller-sa"
```

Reply:
Thank you. There are two service account names in recent versions of the AWS EFS CSI driver Helm chart: `efs-csi-controller-sa` and `efs-csi-node-sa`. I tried to create awareness on the fact I am reusing the same SA for both, by generalising its name as `efs-csi-sa`.